packages/website/src/components/bio.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Bio component that queries for data
+ * with Gatsby's useStaticQuery component
+ *
+ * See: https://www.gatsbyjs.com/docs/use-static-query/
+ */
+
+import * as React from "react";
+import { useStaticQuery, graphql } from "gatsby";
+import { StaticImage } from "gatsby-plugin-image";
+
+const Bio = () => {
+  const data = useStaticQuery(graphql`
+    query BioQuery {
+      site {
+        siteMetadata {
+          author
+          social {
+            twitter
+          }
+        }
+      }
+    }
+  `);
+
+  // Set these values by editing "siteMetadata" in gatsby-config.js
+  const author = data.site.siteMetadata?.author;
+  const social = data.site.siteMetadata?.social;
+
+  return (
+    <div className="bio">
+      <StaticImage
+        className="bio-avatar"
+        layout="fixed"
+        formats={["AUTO", "WEBP", "AVIF"]}
+        src="../images/profile-pic.png"
+        width={50}
+        height={50}
+        quality={95}
+        alt="Profile picture"
+      />
+      {author && (
+        <p>
+          Written by <strong>{author}</strong>
+          {` `}
+          <a href={`https://twitter.com/${social?.twitter || ``}`}>You should follow them on Twitter</a>
```

Comment:
I don't see where this is used but based on the `team.yaml` structure this link shouldn't be prepended with `https://twitter.com/` because `social.twitter` holds a full URL. I won't directly change this because I have no idea where to test the results. :)